### ISSUE TITLE
Cloudflare blocking third-party access

### ISSUE THREAD

**Issue by timeskipp** [tachiyomiorg/tachiyomi-extensions#18626]
### Source information

Shinigami 1.4.34

### Source language

Indonesia

### Steps to reproduce

When you opened the extension it was take so long to loading the content. And the result was blocked by cloudflare. If you try access with web view you can see what's wrong with that.

### Expected behavior

I want to you guys to make that's normally again.

### Actual behavior

I don't want to change my bookmark to the other extension, because it's took to much time for transferred.

### Tachiyomi version

0.14.6

### Android version

Android 8.1

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.
